public/themes/custom/hdbt_subtheme/src/scss/components/webform/_profile_form.scss

```diff
@@ -0,0 +1,34 @@
+.grants-profile-grants-profile {
+  fieldset {
+    border: 0px rgba(0,0,0,0) solid;
+    background: var(--color-black-5);
+    padding: 0 var(--spacing-layout-s) var(--spacing-layout-s);
+    top: calc(-1 * var(--spacing-m));
+    max-width: 714px;
```

Comment:
Mystinen 714px. Olisiko tähän joku muu strategia sopiva tai voisiko tästä tehdä muuttujan?

Reply:
Meillä on jotain näitä kustomiarvoja, jotka tulee siitä, että ei oo lomakkeita muualla. Ne voisi kyl koota johonkin omaan variableensa.